src/family.js

```diff
@@ -14,17 +14,15 @@
  * limitations under the License.
  */
 
-'use strict';
-
-var common = require('@google-cloud/common');
-var createErrorClass = require('create-error-class');
-var is = require('is');
+const common = require('@google-cloud/common');
+const createErrorClass = require('create-error-class');
```

Comment:
You probably don't need this, since you can just extend the Error class.  Feel free to throw in a follow up issue.